recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "hepconvert" %}
-{% set version = "1.3.7" %}
+{% set version = "1.4.0" %}
 {% set python_min = "3.8" %}
```

```suggestion
{% set version = "1.4.0" %}
{% set python_min = "3.9" %}
```

`v1.4.0` is Python `3.9+`.